format-specs/geoparquet.md

```diff
@@ -52,21 +56,21 @@ but have a default geometry used for geospatial operations.
 
 #### version
 
-Version of the GeoParquet spec used, currently 0.4.0
+Version of the GeoParquet spec used, currently 0.5.0-dev
```

Comment:
Something I am wondering about this version number. While it is consistent to use this version here (as that is also used elsewhere, in the schema.json, etc), in practice we also don't really want that anyone would actually create files with this version? 
Do we expect readers to understand that version? (or error for it?)

I am wondering that for readers of the spec, it might be more useful to say here "one of the released versions" instead of actually "currently 0.5.0-dev" ?

Reply:
@jorisvandenbossche - I agree we need to do more cleanup on this.  I think it would be enough for the description of the `version` metadata to say "The version identifier for the GeoParquet specification."  Or we could include language about the "release" version as well.  I don't think we need to repeat the actual version in this description.  In addition, I think the whole "Additional file metadata information" section could be deleted, with the detail instead going in the description column in the table above.

I think the changes in this branch are an incremental improvement, so I'll merge as is.  And then let's keep iterating on improvements.

Reply:
Yes, just removing the version number altogether in this location sounds good

Reply:
Since #159 is about limiting the number of places the version identifier is repeated, I've made this change there.